content/sensu-go/5.20/reference/backend.md

```diff
@@ -417,6 +436,23 @@ sensu-backend start --deregistration-handler /path/to/handler.sh
 deregistration-handler: "/path/to/handler.sh"{{< /highlight >}}
 
 
+| labels     |      |
+-------------|------
+description  | Custom attributes to include with event data for backend assets (e.g. handler and mutator assets).
```

Comment:
I'm wondering if entity would make more sense here, because the event does not get altered with backend entity data. WDYT?
```suggestion
description  | Custom attributes to include with entity data for backend assets (e.g. handler and mutator assets).
```